src/features/weak-references.md

```diff
@@ -217,7 +217,7 @@ After hearing about these new capabilities, it might be tempting to `WeakRef` Al
 
 In general, avoid writing code that depends on the garbage collector cleaning up a `WeakRef` or calling a finalizer at any predictable time — [it can’t be done](https://github.com/tc39/proposal-weakrefs#a-note-of-caution)! Moreover, whether an object is garbage-collectible at all may depend on implementation details, such as the representation of closures, that are both subtle and may differ across JavaScript engines and even between different versions of the same engine.
 
-For example, don’t place important logic in the code path of a finalizer. There’s no way to predict _when_, or even _if_, a given finalizer gets called. It’s best to think of `WeakRef`s and finalizers as **progressive enhancement**: it’s nice if your custom finalizer code runs, but your program should still work without it.
+For example, don’t place important logic in the code path of a finalizer. Finalizer callbacks may not happen immediately after garbage collection, and may not happen in the same order as actual garbage collection. They're useful to perform clean-up in response to garbage-collection, but you can't reliably use them to, say, record meaningful metrics about memory usage.
```

Comment:
If we’re gonna phrase it this way, let’s point to https://web.dev/monitor-total-page-memory-usage/ for this last bit?

Reply:
```suggestion
For example, don’t place important logic in the code path of a finalizer. Finalizer callbacks may not happen immediately after garbage collection, and may not happen in the same order as actual garbage collection. They're useful to perform clean-up in response to garbage-collection, but you can't reliably use them to, say, record meaningful metrics about memory usage. For that use case, see [`measureMemory`](https://web.dev/monitor-total-page-memory-usage/).
```

Reply:
Done